version.h

```diff
@@ -1,5 +1,6 @@
 /* Platform independent version definitions... */
 
+#include "config.h"
```

Comment:
' version.h' file could be installed.
'config.h' file must not be installed so this include will just break installed haders